src/components/tx-flow/flows/SuccessScreen/index.tsx

```diff
@@ -32,17 +35,33 @@ const SuccessScreen = ({ txId }: { txId: string }) => {
   useEffect(() => {
     const unsubFns: Array<() => void> = ([TxEvent.FAILED, TxEvent.REVERTED] as const).map((event) =>
       txSubscribe(event, (detail) => {
-        if (detail.txId === txId) setError(detail.error)
+        if (detail.txId === txId && pendingTx) setError(detail.error)
       }),
     )
 
     return () => unsubFns.forEach((unsubscribe) => unsubscribe())
-  }, [txId])
+  }, [txId, pendingTx])
 
   const onClose = useCallback(() => {
     setTxFlow(undefined)
   }, [setTxFlow])
 
+  const isSuccess = status === undefined
+  const spinnerStatus = error ? SpinnerStatus.ERROR : isSuccess ? SpinnerStatus.SUCCESS : SpinnerStatus.PROCESSING
+
+  let StatusComponent
+  switch (status) {
+    case PendingStatus.PROCESSING:
+    case PendingStatus.RELAYING:
+      StatusComponent = <ProcessingStatus txId={txId} pendingTx={pendingTx} />
+      break
+    case PendingStatus.INDEXING:
+      StatusComponent = <IndexingStatus />
+      break
+    default:
+      StatusComponent = <DefaultStatus error={error} />
```

Comment:
the statusMessage component was packing everything in a blue container (the message: "this tx was confirmed and is now ..."), which according to @TanyaEfremova was not correct. I couldn't figure out a way how to refactor it to accomodate this, so I went for writing as is now in the successScreen.
![grafik](https://github.com/safe-global/safe-wallet-web/assets/693770/dbe099db-33f6-4537-960f-16f00e4b8ad8)

The StatusMessage component is actually no longer used anywhere. If you have ideas how to make it work with the new requirements let me know, otherwise I would actually remove it.

Reply:
I see that you use a similar StatusMessage component in safe creation. The main difference is in the steps and a bit in the UI. The problem here is that if we are in the processing state we need the pendingTx to display the speed up. So it is no longer possible to just pass a `status` and be done with it. If you have some ideas how to abstract this let me know.

Reply:
We could render the `pendingTx` part as a child i.e.

```
<StatusMessage status={status} error={error}>
    {pendingTx && <SpeedUpMonitor txId={txId} pendingTx={pendingTx} modalTrigger={'alertBox'} />}
</StatusMessage>
```

That way we could reuse all the other elements and their styles.

Reply:
How about having it inside `StatusMessage` and only rendering it if `status == PROCESSING` then?